model/user.go

```diff
@@ -225,6 +236,14 @@ func (u *User) AddNotifyProp(key string, value string) {
 	u.NotifyProps[key] = value
 }
 
+func (u *User) GetDisplayName() string {
+	if u.Nickname != "" {
+		return u.Nickname
+	} else {
```

Comment:
Oh yeah, I could see adding that. I wasn't thinking of it at the time since it was just Nickname and then username originally, but with the new First/LastName fields, it makes sense to include that as a second case.